batch/environment.yml

```diff
@@ -11,3 +11,13 @@ dependencies:
   - pylint
   - pytest
   - requests
+  - aiohttp
+  - aiodns
```

Comment:
We’re doing dns resolution?

Reply:
I didn’t see that 
https://github.com/hail-is/hail/pull/5563/files or the cloud MySQL pr’s but I may have missed

Reply:
It's recommended to install it for aiohttp, so I added it: 
https://aiohttp.readthedocs.io/en/stable/#library-installation

Reply:
True, nice catch. I'm still looking where it's used; do you know? I'm still digging through the source code. I see it's used in `resolve.py` and `TCPConnector`, but still need to figure out how those are used.

I'll approve, and open an issue to track that.